bosh-director/spec/unit/deployment_plan/stemcell_spec.rb

```diff
@@ -9,8 +9,8 @@ def make_deployment(name)
     BD::Models::Deployment.make(:name => name)
   end
 
-  def make_stemcell(name, version, os = 'os1')
-    BD::Models::Stemcell.make(:name => name, :operating_system=>os, :version => version)
+  def make_stemcell(name, version, os = 'os1', params = {})
```

Comment:
Ah, didn't catch the `cid`. I don't have strong feelings about params vs separate args in this case, so this seems fine as is.